tests/test_routes.py

```diff
@@ -1,22 +1,84 @@
-# Standard library
 import unittest
 
-# Local
-from webapp.app import app
+from flask_testing import TestCase
 
+from webapp.app import db, create_app
+from webapp.models import CVE, Notice, Status
 
-class TestRoutes(unittest.TestCase):
-    def setUp(self):
-        app.testing = True
-        self.client = app.test_client()
 
-        return super().setUp()
+class TestRoutes(TestCase):
+    def create_app(self):
+        app = create_app()
+
+        app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///../tests/db.sqlite"
+        app.config["TESTING"] = True
+
+        return app
+
+    def setUp(self):
+        db.create_all()
 
-    def test_hello_world(self):
-        self.assertEqual(self.client.get("/hello-world").status_code, 200)
+    def tearDown(self):
+        db.session.remove()
+        db.drop_all()
 
     def test_spec(self):
-        self.assertEqual(self.client.get("/spec").status_code, 200)
+        response = self.client.get("/spec")
+
+        assert response.status_code == 200
 
     def test_docs(self):
-        self.assertEqual(self.client.get("/docs").status_code, 200)
+        response = self.client.get("/docs")
+
+        assert response.status_code == 200
+
+    def test_cve_not_exists(self):
+        response = self.client.get("/cves/CVE-TEST")
+
+        assert response.status_code == 404
+
+    def test_cve(self):
+        cve = CVE(id="CVE-TEST")
+        usn_1 = Notice(id="USN-TEST-1")
+        usn_2 = Notice(id="USN-TEST-2")
+        cve.notices.append(usn_1)
+        cve.notices.append(usn_2)
+        status = Status(
+            cve_id=cve.id,
+            package_name="test_package",
+            release_codename="focal",
+            status="pending",
+        )
+
+        db.session.add(cve)
+        db.session.add(status)
+
+        assert cve in db.session
+        assert status in db.session
```

Comment:
Here I think you're just testing SQLAlchemy, which is out of scope for us. We should be able to trust that if you call `session.add`, then it will be in the session, as part of SQLAlchemy's core function.